client/src/Components/CardContainer/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from 'antd';
+import Card from '../Common/Card';
+import './style.css';
+
+const { Title, Text } = Typography;
+const CardContainer = ({ title, data }) => (
+  <div>
+    <Title level={3} id="container-title">
+      {title}
+    </Title>
+    <div className="container-cards">
+      <ul>
+        {data.length !== 0 ? (
+          data.map((item) => (
+            <li>
+              <Card {...item} />
+            </li>
+          ))
+        ) : (
+          <Text>لم يتم إضافة رياض أطفال.</Text>
+        )}
+      </ul>
+    </div>
+  </div>
+);
+CardContainer.defaultProps = {
+  title: ' ',
+  data: [
+    {
+      title: ' ',
+      rating: ' ',
+      reviewersNo: ' ',
+      location: ' ',
+      minPrice: ' ',
+      maxPrice: ' ',
+      cover: ' ',
+    },
+  ],
```

Comment:
If you call this component like this `<CardContainer />` a default prop with empty values will be rendered not the `Empty` component
```suggestion
  data: []
```